stratum1.yml

```diff
@@ -0,0 +1,13 @@
+- name: CVMFS
+  hosts: cvmfsstratum1servers
+  vars:
+    #cvmfs_srv_device: /dev/sdb
+    eessi_cvmfs_repos_enabled: true
+    # override the default
+    #eessi_cvmfs_server_urls:
+    #  - domain: eessi-hpc.org
+    #    urls:
+    #      - "http://cvmfs.example.org/cvmfs/@fqrn@"
+    #      - "http://cvmfs-s1-rug.eessi-hpc.org/cvmfs/@fqrn@"
```

Comment:
Should we remove the commented out parts, or is there a good reason to keep them?

Reply:
Took this from some example at the Galaxy repo, and wasn't sure why you would need this, so I just removed those lines. Only left the commented line about cvmfs_srv_device, since that's probably useful for anyone installing a Stratum 1.